MAINTAINERS.md

```diff
@@ -0,0 +1,160 @@
+
+A contributor is usually a user as well, but this isn't a hard-and-fast rule. A contributor is also expected to adhere to the [Code of conduct](https://github.com/stryker-mutator/stryker-net/blob/master/CODE_OF_CONDUCT.md) as a user would.
+
+## Maintainer 
+A maintainer has certain "rights" (or "permissions") to the Stryker-net project and other projects under the stryker-mutator organization. These rights come with increased responsibilities.
+
+However, **there is no expectation of a standard of technical ability** to be a maintainer of Stryker-net. This doesn't imply a lack of technical oversight--every pull request will eventually be reviewed.
+
+**If you think you aren't experienced enough to maintain a project like Stryker-net, you are incorrect.** The only requirements are the above responsibilities and a desire to help the project. It bears repeating:
+
+**You don't have to write code to be a maintainer!**
+
+> A maintainer is synonymous with "Collaborator" and/or "Owner" in GitHub parlance.
+
+### The Responsibilities of a Maintainer
+As a maintainer, you are expected to not just "follow" the code of conduct, but embody its values. Your public behavior, whether in the physical or virtual world, reflects upon the project and other maintainers.
+
+> If you don't understand the code of conduct, or why it exists, it is your responsibility to educate yourself. This does not imply the CoC is immutable.
+
+Furthermore, a maintainer is a contributor who contributes regularly, or expresses a desire to do so. That could be every day--but it might be once a week, or even once a month. Your boss doesn't work here; contribute as often as you wish. We are all people with Real Lives, and for many of us, contributing to OSS is just a hobby!
+
+Finally, a maintainer must help define what makes Stryker "Stryker". At a minimum, a maintainer must understand the current definition (if a maintainer is not interested in decision-making). Some of these questions include:
+* What's the scope of Stryker-net?
+* Where should we focus our efforts?
+* What's urgent, what can wait?
+* What can we break? What's off-limits?
+* What user feedback is valuable? What isn't?
+
+As maintainers, _we work together_ to learn about the nature of these questions. If we try hard enough, we even come to some answers!
+
+A maintainer _must_ also have 2FA enabled on their GitHub account.
+
+### The Rights of a Maintainer
+You may choose to do zero or more of these at their discretion:
+1. Merge pull requests
+1. Modify issues (closing, adding labels, assigning them to other maintainers, etc.)
+1. Cancel builds, restart jobs, or otherwise interact with our CI server(s)
+1. CRUD operations on GitHub integrations
+1. Participate in the decision-making process
+
+### About "Owners"
+Some maintainers will have full admin rights to the stryker-mutator organization and/or will have access to publish to npm.
+
+1. Those with publishing access are expected to use npm's 2FA.
```

Comment:
We don't use npm. Not relevant for nuget so can be removed

Reply:
Perhaps it should be relevant for nuget.org?